### PR TITLE
feat(wizard): added drawer

### LINKED ISSUE #4611: Request: Additional Information panel for Wizards
In wizards, we sometimes need to provide a good amount of information to the user about the step they are completing and the context around it. 
Traditionally this would be done with a link to docs (sending user out of context, not focused on the exact step at hand), or tooltips (small) or expanding directions (inline).

I am proposing a pattern in which an informational panel can be optionally added to wizard steps to provide more context. 

image
As seen here, the panel explains a specific tool that users are going to use to register (a new tool) and how to use and configure it. It links out to further documentation but the most crucial information is shown side-by-side with the task the user is doing.

**Application:** Image Builder on console.redhat.com
**Design details:** Initial specs here: https://marvelapp.com/prototype/6g7dja9/screen/84580499

### PR DESCRIPTION
**Pull request by mattnolting**
closes #4611

### CODE REVIEW

**Comment by mattnolting** [comment]
@katierik

File: src/patternfly/components/Drawer/drawer.scss [hunk 1]
```diff
@@ -345,7 +345,6 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
 
 // Modified content children
 .pf-c-drawer__body {
-  min-height: 0;
```
[the remark this answers is not in the document]
Comment (mattnolting) [reply src/patternfly/components/Drawer/drawer.scss:348]: `min-height: 0` was causing the `:last-child` `.pf-c-drawer__body` to collapse. 

Before:
Screen Shot 2022-04-13 at 10 31 29 AM

After:
Screen Shot 2022-04-13 at 10 31 22 AM

[the remark this answers is not in the document]
Comment (mcoker) [reply src/patternfly/components/Drawer/drawer.scss:348]: @mattnolting do you mind adding this declaration back in for the time being so we can test it a little more, and merge this PR? Opened a follow up issue here to follow up - https://github.com/patternfly/patternfly/issues/4797

**Review by mcoker — CHANGES_REQUESTED** [review]

**Review by mcoker — APPROVED** [review]
Great work!!! 🫵😎👍

**Pull request merged**